Sombra.Messaging/Requests/CharityAction/CreateCharityActionRequest.cs

```diff
@@ -18,6 +18,7 @@ public class CreateCharityActionRequest : Request<CreateCharityActionResponse>
         public string CoverImage { get; set; }
         public string ThankYou { get; set; }
         public string UrlComponent { get; set; }
+        public string Logo { get; set; }
```

Comment:
Ook toevoegen in UpdateCharityActionRequest, CharityActionCreatedEvent en CharityActionUpdatedEvent